.github/workflows/assign.yml

```diff
@@ -8,7 +8,9 @@ jobs:
   one:
     runs-on: ubuntu-latest
     steps:
-    - if: github.event.comment.body == 'take'
+    - if: >-
+        github.event.comment.body == 'take' || github.event.comment.body == 'Take'
+        && !github.event.issue.assignee
```

Comment:
Does the logical `OR` (`||`) precede the logical `AND` (`&&`) or the precedences is taken from left to right in `GitHub Actions`?

**Note: I suppose that is the second option.**

Reply:
Added the `()` so future contributors do not need to think about precedence.

(I always assumed that for most languages OR and AND have the same precedence )

Reply:
I think in general `and` has priority over `or`

They can't have the same otherwise `False and True or True` is undefined